multiversx_sdk/core/code_metadata_test.py

```diff
@@ -1,8 +1,39 @@
 
 
+import pytest
+
 from multiversx_sdk.core.code_metadata import CodeMetadata
 
 
+def test_code_metadata_new_from_bytes():
+    metadata = CodeMetadata.new_from_bytes(bytes([0x05, 0x00]))
+    assert metadata.upgradeable == True
+    assert metadata.readable == True
+    assert metadata.payable == False
+    assert metadata.payable_by_contract == False
+
+    metadata = CodeMetadata.new_from_bytes(bytes([0x05, 0x06]))
+    assert metadata.upgradeable == True
+    assert metadata.readable == True
+    assert metadata.payable == True
+    assert metadata.payable_by_contract == True
+
+    metadata = CodeMetadata.new_from_bytes(bytes([0x05, 0x00]))
+    assert metadata.upgradeable == True
+    assert metadata.readable == True
+    assert metadata.payable == False
+    assert metadata.payable_by_contract == False
```

Comment:
This is duplicated, I think you meant:
```py
metadata = CodeMetadata.new_from_bytes(bytes([0x00, 0x05]))
```

Can be fixed in the next PR.

Reply:
Good catch. Fixed (mirroring the test below).